kmk/preload_imports.py

```diff
@@ -1,45 +0,0 @@
-# Welcome to RAM and stack size hacks central, I'm your host, klardotsh!
```

Comment:
oh thank zeus this file is dead

have you tested that this is backwards compatible to M4? M4 was originally where this hack had to be developed

Reply:
It's running on the live code in this PR on Nyquist (wired split) luddite (also M4) and corne BLE split nice!nano on custom firmware. Full test suite.